stockfish/models.py

```diff
@@ -278,6 +279,53 @@ def get_evaluation(self) -> dict:
                         }
             elif splitted_text[0] == "bestmove":
                 return evaluation
+    
+    def get_top_moves(self, num_top_moves: int) -> List[dict]:
```

Comment:
please, add this function to documentation in https://github.com/zhelyabuzhsky/stockfish/blob/master/README.md